include/ssd.h

```diff
@@ -87,6 +87,7 @@ bool ssd_is_button(enum ssd_part_type type);
 bool ssd_part_contains(enum ssd_part_type whole, enum ssd_part_type candidate);
 
 /* TODO: clean up / update */
+bool has_ssd(struct view *view);
```

Comment:
If this has to be public, it should be prefixed with `ssd_`.

Reply:
It's also misnamed, because the function indicates whether the view *wants* SSD, not whether it has them.